Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerCommandCollectionViewCell_Tests.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerCommandCellView_Tests: XCTestCase {
+    func test_defaultAppearance() {
+        let view = ChatMessageComposerCommandCellView()
+        view.command = .init(name: "Hello", description: "what", set: "is this", args: "WHAT")
```

Comment:
This is a strict NO-GO. None of these words is related to StarWars.

Reply:
Yes this commit had old code, forgot to edit it 👍  I've pushed the changed, re-review the commit please